changelog.md

```diff
@@ -3,6 +3,11 @@ This file contains all the notable changes done to the Ballerina STAN package th
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+- [Fix historical message replay not working as intended.](https://github.com/ballerina-platform/ballerina-standard-library/issues/1420)
+
```

Comment:
As per the discussion on changelog, the changes which we do for current release should be in the unreleased section until we complete the release. @shafreenAnfar wdyt?

Reply:
This iss in the unreleased section already. What do you mean?

Reply:
<img width="528" alt="Screenshot 2021-06-11 at 16 30 57" src="https://user-images.githubusercontent.com/39673615/121676588-6caaae00-cad2-11eb-8987-1a0be7745c03.png">
O.o

Reply:
Sorry my mistake :+1: